.github/workflows/ruby.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['3.2', '3.3']
+        ruby: ['3.3', '3.4']
```

Comment:
Reserves is currently deployed on 3.2, so we should keep testing on  3.2 until we deploy on a newer version.

Reply:
Now on 33.